pandas/tests/frame/methods/test_compare.py

```diff
@@ -180,3 +180,28 @@ def test_compare_unaligned_objects():
         df1 = pd.DataFrame(np.ones((3, 3)))
         df2 = pd.DataFrame(np.zeros((2, 1)))
         df1.compare(df2)
+
+
+def test_compare_suffixes():
+    # GH
+    df1 = pd.DataFrame(
+        {"col1": ["a", "b", "c"], "col2": [1.0, 2.0, np.nan], "col3": [1.0, 2.0, 3.0]},
+        columns=["col1", "col2", "col3"],
```

Comment:
not necessary, dict keys define columns already

Reply:
makes sense

Reply:
Done